sql/parser/builtins.go

```diff
@@ -248,6 +248,33 @@ var builtins = map[string][]builtin{
 		return DString(strings.Replace(s, from, to, -1)), nil
 	}, DummyString)},
 
+	"translate": {stringBuiltin3(func(s, from, to string) (Datum, error) {
+		const deletionRune = rune(0)
```

Comment:
Is `rune(0)` an invalid rune? Can't I generate that using `e'\x00'` for the `to` string?